### DISCUSSION TITLE
Accept URL parameters to either trigger an action or fill in a form

### DISCUSSION THREAD

**Discussion by sxwebster** [ToolJet/ToolJet#3424] [Q&A]
I have an application who's only method of sending information elsewhere is via GET request. Ie calling a URL. Tableau and PowerBI both suffer from this. 

Can Tooljet extract a URL parameter into a field if the application is called with a parameter in the URL? Can the same be used to trigger an action such as updating a DB row?

Ie if the URL includes &Name=Boris can Tooljet then fill the name field when the page loads with "Boris"? If so, how is this achieved?

**Comment by withshubh** [comment]
Hey @sxwebster 👋 
The `urlparams` variable is exposed under globals. It will include the query params. 
You can access them using `{{globals.urlparams.abc}}`

![image (6)](https://user-images.githubusercontent.com/25361949/176542093-888afff2-0ed8-452c-9a11-1803ea7da3d3.png)

**Comment by sxwebster** [comment]
@withshubh thank you very much
